src/logger/index.ts

```diff
@@ -524,15 +524,14 @@ class Logger {
         } else {
           const sanitizedFileName = path.basename(requestedFileName)
           if (!validFileNames.includes(sanitizedFileName)) {
-            res.status(400).send('Invalid log file specified')
+            res.status(400).json('Invalid log file specified')
```

Comment:
this is res.json, but we are sending a string instead of a json object. sam is out, so i'll push the fix soon